app/src/main/res/font/blinker_bold.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<font-family xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
```suggestion
<font-family xmlns:app="http://schemas.android.com/apk/android"
```

Ran into an `unknown font` problem with res-auto. Look up documentation and they use android instead.